ibc-testkit/src/fixtures/clients/tendermint.rs

```diff
@@ -172,9 +174,28 @@ pub fn dummy_ics07_header() -> Header {
     }
 }
 
+#[cfg(feature = "serde")]
+pub fn test_serialization_roundtrip<T>(json_data: &str)
```

Comment:
I wasn't aware of this function. It seems pretty general and useful. Can we put it somewhere like `utils.rs`?

Reply:
[4211e04](https://github.com/cosmos/ibc-rs/pull/1233/commits/4211e04aa1124e6be073e32392656c4e614f9fda)